test/apps/iodemo/io_demo.cc

```diff
@@ -1845,13 +1845,9 @@ class DemoClient : public P2pDemoCommon {
             op_info[op_id].num_iters = 0;
         }
 
-        log << ", active:" << _active_servers.size();
-
-        if (opts().window_size == 1) {
-            log << ", latency:" << latency_usec << " usec";
-        }
-
-        log << ", buffers:" << _data_buffers_pool.allocated();
+        log << ", active:" << _active_servers.size()
+            << ", latency:" << latency_usec << " usec"
```

Comment:
should `latency_usec` be divided by `windows_size`?

Reply:
Latency_usec is calculated in this way: https://github.com/openucx/ucx/blob/master/test/apps/iodemo/io_demo.cc#L1798
Since it always use ```wait_for_completion(0)```, so all the previous operation should have been completed before calcuating the current average latency.